h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -286,11 +288,7 @@ public void importFiles(String path, String pattern, ArrayList<String> files, Ar
       I[Value.NFS].importFiles(path, pattern, files, keys, fails, dels);
     } else if ("http".equals(scheme) || "https".equals(scheme)) {
       try {
-        java.net.URL url = new URL(path);
-        Key destination_key = Key.make(path);
-        java.io.InputStream is = url.openStream();
-        UploadFileVec.ReadPutStats stats = new UploadFileVec.ReadPutStats();
-        UploadFileVec.readPut(destination_key, is, stats);
+        Key destination_key = FrameUtils.eagerLoadFromHTTP(path);
```

Comment:
Semantics is not being changed here. When user calls `h2o.import` we load data eagerly, but that is expected in this call